boards/wiznet/w5500_evb_pico/board.yml

```diff
@@ -1,6 +1,6 @@
 board:
   name: w5500_evb_pico
-  full_name: W5500 Evaluation Pico
+  full_name: W5500-EVB-Pico
```

Comment:
commits should be squashed

Reply:
Did not notice it's different boards

Reply:
Thank you for confirmation. I got it. I'l leave it.